### PR TITLE
Branch update DG for AddCommand

### PR DESCRIPTION
**Pull request by TanWenCong**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -168,6 +168,46 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 ## **Implementation**
 
 This section describes some noteworthy details on how certain features are implemented.
+### \[Insert Numbering\] Add feature
+The Edit feature is facilitated by `LogicManager`. The `AddCommandParser` parses the command arguments, and returns
```
Comment (bensohh) [inline docs/DeveloperGuide.md]: I think this should be The Add Feature is facilitated, just a small typo.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -168,6 +168,46 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+an `AddCommand` that is executed by the `LogicManager`.
+
+This feature allows the user to add a new Customer.
+
+**Below is a sample usage and how the add sequence behaves at each step.**
+
+1. User chooses the Customer he/ she wants to add and enters the command `add n/Bob p/12345678 e/johnd@example.com r/5000 t/GOLD t/MEMBER`
```
Comment (bensohh) [inline docs/DeveloperGuide.md]: Just a small formatting issue, perhaps the space can be removed between he/she

**Review by bensohh — APPROVED** [review]
LGTM! Just a few errors/formatting to fix! Cheers! 🧋

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (TanWenCong) [reply docs/DeveloperGuide.md]: Resolved, thanks for spotting it!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (TanWenCong) [reply docs/DeveloperGuide.md]: Resolved, thanks for spotting it!

**Pull request merged**